index.html

```diff
@@ -12,19 +12,19 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap" rel="stylesheet">
 </head>
 <body>
-    <main class="container">
-        <header class="navbar">
-            <img class="toolbar" src="img/shapeheader.png" alt="Shape1">
-            <img class="rectangle" src="img/menurectangle.png" alt="Rectangle">
+        <header>
+            <nav class="navbar">
+                <a href="index.html"><img class="toolbar" src="img/shapeheader.png" alt="Shape1"></a>
+                <img class="rectangle" src="img/menurectangle.png" alt="Rectangle">
+            </nav>
         </header>
         <section class="headline">
             <h1>Marco Antonio Hincapié Montes</h1>
             <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
             <div class="collaboration_buttom">
-                <h6>Start collaboration</h6>
+                <button>Start collaboration</button>
```

Comment:
- [ ] Great job so far 👍 You are almost there. Kindly add the various button interactions as shown in the template for a better user interface 
- Button interactions;
![Annotation 2022-03-17 192100](https://user-images.githubusercontent.com/85113567/158870160-c798d75b-1a8d-480f-af86-ef50194cea3a.jpg)

Reply:
Hi @Kingstalux,
I already understand what you are requesting, but it is not necessary for this part of the project.
For this part of the project the only thing that I have to do is the first part of the main page and the navbar for the mobile version.
Can you clarify this to me, please?
Best regards.
Marco Hincapié

Reply:
Hello @AntonioHincapie , 
You are right but in the first part of the main page there is a button and it is a major requirement to implement all button interactions if a button is present in the template

Reply:
Hi @Kingstalux,
Yes, I see that template but that part doesn't bellow to the main mobile version, for being honest I didn't learn how to do that feature. If it is mandatory, can you please send me material to do this feature?

Reply:
![Annotation 2022-03-17 195603](https://user-images.githubusercontent.com/85113567/158875834-cf5e0a11-32f1-44bd-b7e4-a1fa07642259.jpg)
This is your site and I've marked the button where you've to implement the button interactions. You can learn more about button interactions [here](https://www.w3schools.com/howto/howto_css_animate_buttons.asp).